mcafee_atd/plugin.spec.yaml

```diff
@@ -643,3 +643,53 @@ actions:
         description: Returns true if found information about analyzer profiles
         type: boolean
         required: true
+  get_report:
+    title: Get Report
+    description: Download the analysis report file for given parameter
+    input:
+      id:
+        title: ID
+        description: Task ID or job ID or MD5 value for which will be prepared analysis report
+        type: string
+        required: true
+        example: 13
+      type_id:
+        title: Type ID
+        description: Type of given ID parameter, default value is MD5
+        type: string
+        required: false
+        default: MD5
+        example: TASK ID
+        enum:
+          - MD5
+          - TASK ID
+          - JOB ID
+      report_type:
+        title: Report Type
+        description: Type of file prepared analysis report
+        type: string
+        required: false
+        default: JSON
+        example: HTML
+        enum:
+          - JSON
```

Comment:
If we always return JSON as `report` OUTPUT, we can remove this one.

Reply:
We are not return always JSON as `report`. It is possible only when we choose JSON in this input. For HTML, PDF and others we don't returned JSON as `report` (we can't do it for other cases, because response from server is not JSON).